libs/completer/completer_test.go

```diff
@@ -0,0 +1,104 @@
+package completer
```

Comment:
This is not a generic completer but one that operates on filers. Could you move the pkg under `libs/filer`?

Reply:
Moved!